app/javascript/css/tailwind.scss

```diff
@@ -24,103 +24,76 @@
     @apply flex mb-4 items-center;
 }
 
+#form-grid {
+    display: grid;
+    grid-template-columns: 0.8fr 0.2fr;
+    grid-template-rows: 1fr;
+    grid-template-areas: "form-name form-button";
+}
+
 #form-name {
-    @apply text-left w-full text-grey-darkest;
+    grid-area: form-name;
+    display: grid;
+    @apply align-text-bottom text-left w-full text-grey-darkest font-sans font-thin mt-5;
 }
 
 #form-button {
-    @apply text-right;
+    grid-area: form-button;
+    display: grid;
+    @apply text-right font-thin;
+}
+
+.add-field {
+    @apply mt-4;
 }
 
 .add-field-placeholder {
-    @apply shadow appearance-none border rounded w-full py-2 px-3 mr-4 text-grey-darker;
+    @apply shadow appearance-none border rounded w-full py-2 px-3 mr-4 text-grey-darker font-sans font-thin;
 }
 
 .button {
-    @apply flex-no-shrink p-2 border-2 rounded text-teal border-teal;
+    @apply flex-no-shrink p-2 my-2 border-2 rounded text-grey-darker border-grey-darker font-sans font-thin;
 }
 
 .button:hover {
-    @apply text-white bg-teal ;
+    @apply text-white bg-grey-darkest;
 }
 
 .grid-container {
     display: grid;
-    height: 90vh;
-    grid-template-columns: 0.2fr 0.9fr 0.3fr;
+    grid-template-columns: 0.2fr 0.7fr 0.3fr;
     grid-template-rows: 1fr;
     grid-template-areas: "sidebar content shifts";
 }
 
-#header {
+#sidebar { 
+    grid-area: sidebar;
     display: grid;
-    height: 15vh;
-    grid-template-areas: "header-left header-right";
-    grid-template-columns: 0.8fr 0.2fr;
-    grid-template-rows: 1fr;
-    @apply bg-grey-darkest px-16 py-6 ;
-    h1 {
-        @apply font-semibold text-4xl text-white tracking-tight w-full flex-grow ;
-    }
+    @apply font-sans antialiased h-screen;
 }
 
-#header-left {
-    align-content: start;
-    font-size: 32px;
-    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
+.sidebar-background {
+    @apply bg-grey-darkest relative h-full min-h-screen;
 }
 
-#header-right{
-    align-content: start;
-}
+// .sidebar-button {
+//     @apply block text-center shadow-light py-6 border-l-4 border-transparent;
+// }
 
-.header-button {
-    @apply py-4 px-8 bg-blue text-white text-center my-4 mx-4 align-middle no-underline rounded;
-}
+// .sidebar-button:xl {
+//     @apply flex items-center text-left shadow-none py-2 px-4 ;
+// }
 
+// .sidebar-button:hover {
+//     @apply bg-black;
+// }
```

Comment:
Please remove comments like this

Reply:
Ok

Reply:
If you read these you'll notice I was attempting to extract css classes but failing (per your comments below). So left here in case there was an easy fix.

Reply:
They are removed